.eslintrc.json

```diff
@@ -120,7 +124,7 @@
                         "allowConditionalTypes": "always",
                         "allowMappedTypes": "always",
                         "allowCallbacks": "always",
-                        "allowLiterals": "never"
+                        "allowLiterals": "in-intersections"
```

Comment:
Reverted interfaces and changed the rule for "no-empty-interface". We need to used "allowLiterals": "in-intersections", because we have `export type IProportionalAggregatorFn = ((origin: IProportionalAggregatorOrigin, properties?: IProportionalDonutContentAggregatorProperties) => string) & {
    aggregatorType: string;
};`, which used type alias.